### DISCUSSION TITLE
MSSQL Error on migration to 0.91.1 [May 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2280] [Discord Threads]
> 🙋‍♂️ **LatinoFlayva**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@186479783092289537>    
#### Help us by raising a issut on github with relevant details please. https://github.com/nocodb/nocodb/issues  
#  
> 🙋‍♂️ **LatinoFlayva**  
#### I have done this and they have put a fix in, is there any way i can edit the install to get it happening now, i can't see the source anywhre on my server, thye must pull the db definitions remotely  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/980695372789870622)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
